client/commons/constants.js

```diff
@@ -3,6 +3,10 @@ export const ACTION_TYPE = {
     SET_USER: 'SET_USER'
 }
 
+const USER = 'user';
+
 export const ROUTE_PATH = {
-    DOGS: '/dogs'
+    DOGS: '/dogs',
+    USER_PROFILE: `/${USER}/profile`,
+    USER_ORDERS: `/${USER}/orders`,
```

Comment:
just thinking if we should just stick to duplicated `/user/profile` and `user/orders`...